### ISSUE TITLE
Internal compiler error gfortran

### ISSUE THREAD

**Issue by hsorby** [OpenCMISS/iron#181]
When compiling Iron on Ubuntu 18.04 I am getting the following error:

```
/home/cmiss/tmp/opencmiss/src/iron/src/fieldml_output_routines.F90:182:0:

       FIELDML_OUTPUT_IMPORT_FML = Fieldml_AddImport( FML_HANDLE, IMPORT_INDEX, cchar(REMOTE_NAME), cchar(REMOTE_NAME) )

internal compiler error: Segmentation fault
```
this is the gfortran compiler version that comes with Ubutnu 18.04

```
GNU Fortran (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
```

**Comment by sgenie68** [comment]
You need to get GCC updated to 8+ version

Gene

-- 
Scientific Software Specialist
Auckland Bioengineering Institute
The University of Auckland
e: g.soudlenkov@auckland.ac.nz
p: +64 9 3737599 ext 89834 c: +64 21 840 825
w: https://www.auckland.ac.nz/en/abi.html

**Comment by hsorby** [comment]
I am also getting the same error on gfortran version 8.3

**Comment by sgenie68** [comment]
Are you sure you used 8.3? 

Cheers
Gene

**Comment by sgenie68** [comment]
Hugh, where did you try compiling it? bioeng49.uoa.auckland.ac.nz had 
gcc/gfortran reverted back to v7 for some reason - I set up aliases for 
cmiss user to point to 8.3

Cheers,
Gene

-- 
Scientific Software Specialist
Auckland Bioengineering Institute
The University of Auckland
e: g.soudlenkov@auckland.ac.nz
p: +64 9 3737599 ext 89834 c: +64 21 840 825
w: https://www.auckland.ac.nz/en/abi.html

**Comment by chrispbradley** [comment]
I've got 8.3 on my Fedora 29 system and 8.3 works fine and doesn't give a compiler internal error.

**Comment by hsorby** [comment]
cmiss user on bioeng49 /home/cmiss/tmp directory.

**Comment by sgenie68** [comment]
Yeah - I requested Tiong to set gcc-8 as a default gcc but yesterday it 
was reset back to gcc-7. But cmniss user now has aliases set to gcc-8

Gene

-- 
Scientific Software Specialist
Auckland Bioengineering Institute
The University of Auckland
e: g.soudlenkov@auckland.ac.nz
p: +64 9 3737599 ext 89834 c: +64 21 840 825
w: https://www.auckland.ac.nz/en/abi.html

**Comment by hsorby** [comment]
So the update-alternatives was not done correctly and gfortran wasn't updated to match the gcc version.  I have now corrected this and things will now hopefully carry on as before.  I am currently waiting to see if this has corrected the problem.  Which I think it will.

And please do let us know when you are upgrading software on the build slaves as there are repercussions of these changes.
